content/sql/concepts/primary-keys/primary-keys.md

```diff
@@ -48,3 +48,33 @@ CREATE TABLE students (
 When the primary key for one table appears in a different table, it is called a foreign key.
 
 Why is this important? The most common types of joins will be joining a foreign key from one table with the primary key from another table. For instance, when we join the `orders` table and the `customers` table, we join on the `customer_id` column, which is a foreign key in `orders` and the primary key in `customers`.
+
+## Composite Keys
+
+Having one primary key per table is sometimes not enough to uniquely identify a row; in such cases we would use more than one column, or **composite keys**. This requirement should be detected during the designing phase of a database. 
```

Comment:
```suggestion
Sometimes, having one primary key per table is not enough to uniquely identify a row. In such cases, multiple columns would work as composite keys for the table. This requirement should be detected during the designing phase of a database.
```

Reply:
No need to **emphasize** here; only the concept/term matching the `Title` should be bolded once, near the top of the file.